services/core-web/src/components/noticeOfWork/NOWStatusIndicator.js

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable */
-import React from "react";
+import React, { useState } from "react";
 import { connect } from "react-redux";
 import { PropTypes } from "prop-types";
+import { isEmpty } from "lodash";
 import { Alert, Badge, Tooltip } from "antd";
 import { withRouter } from "react-router-dom";
 import CustomPropTypes from "@/customPropTypes";
-import { getNoticeOfWork } from "@common/selectors/noticeOfWorkSelectors";
+import { getNoticeOfWork, getApplictionDelay } from "@common/selectors/noticeOfWorkSelectors";
```

Comment:
`getApplictionDelay` typo :(

Reply:
Updated all spelling mistakes in my current PR